### PR TITLE
docs: Update configuration.asciidoc

### PR DESCRIPTION
**Pull request by leahleahy**

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -215,8 +214,8 @@ Distributed tracing is enabled by default, however you can use this configuratio
 * *Default:* `[]`
 
 This option can be set to an array containing one or more Strings or RegExp objects and determines which origins should be monitored as part of distributed tracing.
-More specifically, this option is consulted when the agent is about to add the distributed tracing HTTP header (`traceparent`) to a request.
-Please note that each item in the array should be a valid URL containing the origin (other parts of the url are ignored) or a RegExp object. If an item in the array is a String, an exact match will be performed. If it's a RegExp object, its test function will be called with the request origin. 
+This option is consulted when the agent is about to add the distributed tracing HTTP header (`traceparent`) to a request.
+Please note that each item in the array should be a valid URL containing the origin (other parts of the url are ignored) or a RegExp object. If an item in the array is a string, an exact match will be performed. If it's a RegExp object, its test function will be called with the request origin.
```
Comment (leahleahy) [inline docs/configuration.asciidoc:218]: String is inconsistently capitalized. Should it always be capitalized?

File: docs/configuration.asciidoc [hunk 2]
```diff
@@ -234,7 +233,7 @@ var options = {
 
 When distributed tracing is enabled, this option can be used to propagate the https://www.w3.org/TR/trace-context/#tracestate-header[tracestate]
 HTTP header to the configured origins. Before enabling this flag, make sure to change your <<server-configuration, server configuration>> to avoid
-Cross-Origin Resource Sharing errors.
+Cross-Origin Resource Sharing errors.
```
Comment (leahleahy) [inline docs/configuration.asciidoc:236]: Should cross origin be capitalized? Or resource sharing?

**Comment by bmorelli25** [comment]
Jenkins, run the tests please

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by vigneshshanmugam** [comment]
@bmorelli25 CI Lint would fail without a proper commit message and subject.

File: docs/configuration.asciidoc [hunk 3]
```diff
@@ -115,10 +114,10 @@ You must use the query bar to filter for a specific environment in versions prio
 
 
 Set the verbosity level for the agent.
-Note that this does not have any influence on the types of errors that are sent to the APM Server.
-This option is useful when you want to report an issue with the agent to us.
+This does not have any influence on the types of errors that are sent to the APM Server. Use this option to report an issue with the agent to us.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:117]:
```suggestion
This does not have any influence on the types of errors that are sent to the APM Server. This option is useful when you want to report an issue with the agent to us.
```

File: docs/configuration.asciidoc [hunk 4]
```diff
@@ -204,7 +203,7 @@ documentation for more details.
 * *Type:* Boolean
 * *Default:* `true`
 
-Distributed tracing is enabled by default, however you can use this configuration to disable it.
+The distributed tracing is enabled by default, but this configuration can be used to disable it.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:206]:
```suggestion
Distributed tracing is enabled by default. Use this option to disable it.
```

File: docs/configuration.asciidoc [hunk 5]
```diff
@@ -246,7 +245,7 @@ Throttle the number of events sent to APM Server.
 [[events-limit]]
 ==== `eventsLimit`
 
-By default, the agent can only send up to `80` events every `60000` milliseconds (one minute).
+The agent can only send up to `80` events every `60000` milliseconds (one minute) by default.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:248]:
```suggestion
By default, the agent can only send up to `80` events every `60000` milliseconds (one minute).
```

File: docs/configuration.asciidoc [hunk 6]
```diff
@@ -258,7 +257,7 @@ By default, the agent can only send up to `80` events every `60000` milliseconds
 * *Type:* Number
 * *Default:* `1.0`
 
-A number between `0.0` and `1.0` that specifies the sample rate of transactions, by default all transactions are sampled.
+A number between `0.0` and `1.0` that specifies the sample rate of transactions. By default all transactions are sampled.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:260]:
```suggestion
A number between `0.0` and `1.0` that specifies the sample rate of transactions. By default, all transactions are sampled.
```

File: docs/react-integration.asciidoc [hunk 7]
```diff
@@ -66,7 +66,7 @@ class App extends React.Component {
 }
 ----
 
-NOTE: `ApmRoute` only instruments the route if component property is provided, in other cases, e.g. using `render` or `children` properties, 
+NOTE: `ApmRoute` only instruments the route if `component` property is provided, in other cases, e.g. using `render` or `children` properties,
```
Comment (bmorelli25) [inline docs/react-integration.asciidoc:69]:
```suggestion
NOTE: `ApmRoute` only instruments the route if component property is provided, in other cases, e.g. using `render` or `children` properties, 
```

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
Jenkins, run the tests please

**Review by bmorelli25 — APPROVED** [review]
Pending the doc build, LGTM

**Pull request merged**